mod/notifications/views/default/notifications/subscriptions/forminternals.php

```diff
@@ -6,40 +6,30 @@
  */
 
```

Comment:
This view desperately needs breaking into subviews. I'd rather we did that first (far easier to verify correctness).